### ISSUE TITLE
Documentation bug: HTML for .basic.modal doesn't match actual HTML

### ISSUE THREAD

**Issue by brendon** [Semantic-Org/Semantic-UI#3428]
In the Modal documentation, the HTML displayed for a basic modal:

``` html
<div class="ui basic modal">
  <i class="close icon"></i>
  <div class="header">
    Archive Old Messages
  </div>
  <div class="image content">
    <div class="image">
      <i class="archive icon"></i>
    </div>
    <div class="description">
      <p>Your inbox is getting full, would you like us to enable automatic archiving of old messages?</p>
    </div>
  </div>
  <div class="actions">
    <div class="two fluid ui inverted buttons">
      <div class="ui red basic inverted button">
        <i class="remove icon"></i>
        No
      </div>
      <div class="ui green basic inverted button">
        <i class="checkmark icon"></i>
        Yes
      </div>
    </div>
  </div>
</div>
```

is actually different from the HTML that is executed upon:

``` html
<div class="ui small basic test modal">
    <div class="ui icon header">
      <i class="archive icon"></i>
      Archive Old Messages
    </div>
    <div class="content">
      <p>Your inbox is getting full, would you like us to enable automatic archiving of old messages?</p>
    </div>
    <div class="actions">
      <div class="ui red basic cancel inverted button">
        <i class="remove icon"></i>
        No
      </div>
      <div class="ui green ok inverted button">
        <i class="checkmark icon"></i>
        Yes
      </div>
    </div>
  </div>
```

Not a huge deal, but I wanted to emulate that particular look and had to delve into the actual source code of the page to find the code.

**Comment by coolblabla** [comment]
+1  oh~ you saved my time!

**Comment by Banandrew** [comment]
Fixed with Semantic-Org/Semantic-UI-Docs#247.
